src/actions/configuration/first-time-configuration-action.php

```diff
@@ -104,7 +123,17 @@ public function set_site_representation( $params ) {
 	 * @return object The response object.
 	 */
 	public function set_social_profiles( $params ) {
-		$failures = $this->social_profiles_helper->set_organization_social_profiles( $params );
+		$failures   = $this->social_profiles_helper->set_organization_social_profiles( $params );
+		$old_values = $this->get_old_values( \array_keys( $this->social_profiles_helper->get_organization_social_profile_fields() ) );
+
+		/**
+		 * Action: 'wpseo_post_update_social_profiles' - Allows for Hiive event tracking.
+		 *
+		 * @param array The new values of the options.
+		 * @param array The old values of the options.
+		 * @param array The options that failed to be saved.
+		 */
+		\do_action( 'wpseo_post_update_social_profiles', $params, $old_values, $failures );
```

Comment:
* You used in the add_action `wpseo_ftc_post_update_social_profiles` elsewhere!
* Maybe signal this as `@internal`? Or create documentation PR?

Reply:
Done